doc/releases/changelog-dev.md

```diff
@@ -242,62 +242,73 @@
+  at a cost of needing to perform a greater number of device executions. Circuit cutting is enabled
+  by decorating a QNode with the `@qml.cut_circuit` transform.
+  [(#2107)](https://github.com/PennyLaneAI/pennylane/pull/2107)
+  [(#2124)](https://github.com/PennyLaneAI/pennylane/pull/2124)
+  [(#2153)](https://github.com/PennyLaneAI/pennylane/pull/2153)
+  [(#2165)](https://github.com/PennyLaneAI/pennylane/pull/2165)
+  [(#2158)](https://github.com/PennyLaneAI/pennylane/pull/2158)
+  [(#2169)](https://github.com/PennyLaneAI/pennylane/pull/2169)
+  [(#2192)](https://github.com/PennyLaneAI/pennylane/pull/2192)
+  [(#2216)](https://github.com/PennyLaneAI/pennylane/pull/2216)
+  [(#2168)](https://github.com/PennyLaneAI/pennylane/pull/2168)
+  [(#2231)](https://github.com/PennyLaneAI/pennylane/pull/2231)
+  [(#2234)](https://github.com/PennyLaneAI/pennylane/pull/2234)
+  [(#2244)](https://github.com/PennyLaneAI/pennylane/pull/2244)
+  [(#2251)](https://github.com/PennyLaneAI/pennylane/pull/2251)
+  [(#2265)](https://github.com/PennyLaneAI/pennylane/pull/2265)
+  [(#2254)](https://github.com/PennyLaneAI/pennylane/pull/2254)
+  [(#2260)](https://github.com/PennyLaneAI/pennylane/pull/2260)
+  [(#2257)](https://github.com/PennyLaneAI/pennylane/pull/2257)
+  [(#2279)](https://github.com/PennyLaneAI/pennylane/pull/2279)
   
-  - `WireCut` operations that do not lead to a disconnection are now being removed.
-    [(#2260)](https://github.com/PennyLaneAI/pennylane/pull/2260)
+  The example below shows how a three qubit circuit can be run on a two qubit device:
   
-  - Circuit cutting now remaps the wires of fragment circuits to match the available wires on the
-    device.
-    [(#2257)](https://github.com/PennyLaneAI/pennylane/pull/2257)
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.cut_circuit
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RX(x, wires=0)
+      qml.RY(0.9, wires=1)
+      qml.RX(0.3, wires=2)
+
+      qml.CZ(wires=[0, 1])
+      qml.RY(-0.4, wires=0)
+
+      qml.WireCut(wires=1)
+
+      qml.CZ(wires=[1, 2])
+
+      return qml.expval(qml.grouping.string_to_pauli_word("ZZZ"))
+  ```
+  
+  Instead of executing the circuit directly, it will be partitioned into smaller fragments
+  according to the `WireCut` locations, and each fragment executed multiple times. Combining the
+  results of the fragment executions will recover the expected output of the original uncut circuit.
+  
+    ```pycon
+  >>> x = np.array(0.531, requires_grad=True)
+  >>> circuit(0.531)
+  0.47165198882111165
+  ```
 
-  - The documentation has been polished.
-    [(#2279)](https://github.com/PennyLaneAI/pennylane/pull/2279)
+  Circuit cutting support is also differentiable:
+  
+  ```pycon
+  >>> qml.grad(circuit)(x)
+  -0.276982865449393
+  ```
+  
+  For more details on circuit cutting, check out the
+  [qml.transforms.qcut](https://pennylane.readthedocs.io/en/stable/code/qml_transforms_qcut.html)
```

Comment:
I can't seem to find such a link on the latest doc page, is it available already?

Minor: would we like to pin to the "latest" doc link here such that it's live from the get-go?

Reply:
Yes good point - it's also not yet available on https://pennylane.readthedocs.io/en/lates/code/qml_transforms_qcut.html because `qml_transforms_qcut` is only added in the `qcut_tidy` branch which will merge into the release candidate branch :thinking:

Reply:
Oh right! We could simply just merge the release candidate branch into master once `qcut_tidy` has been merged into rc. That way we can make sure the link is okay.

Reply:
That's a good idea!